--- blacklist.js	2016-02-08 11:14:47.000000000 +0000
+++ blacklist.js	2016-02-08 11:16:18.000000000 +0000
@@ -16,6 +16,8 @@
   /node_modules[/\\]react[/\\]dist[/\\].*/,
   'node_modules/react/lib/React.js',
   'node_modules/react/lib/ReactDOM.js',
+  /web\/.*/,
+  /windows\/.*/,

   // For each of these fbjs files (especially the non-forks/stubs), we should
   // consider deleting the conflicting copy and just using the fbjs version.
